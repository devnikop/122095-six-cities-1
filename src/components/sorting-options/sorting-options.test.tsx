import * as React from 'react';
import * as renderer from 'react-test-renderer';

import SortingOptions from './sorting-options';

it(`SortingOptions renders correctly`, () => {
  const tree = renderer
    .create(<SortingOptions
      handleSelectChange={jest.fn()}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
