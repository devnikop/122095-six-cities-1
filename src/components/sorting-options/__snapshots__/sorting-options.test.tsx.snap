// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortingOptions renders correctly 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
>
  <span
    className="places__sorting-caption"
  >
    Sort by
  </span>
  <select
    className="places__sorting-type"
    id="places-sorting"
    onChange={[MockFunction]}
  >
    <option
      className="places__option"
      value="popular"
    >
      Popular
    </option>
    <option
      className="places__option"
      value="to-high"
    >
      Price: low to high
    </option>
    <option
      className="places__option"
      value="to-low"
    >
      Price: high to low
    </option>
    <option
      className="places__option"
      value="top-rated"
    >
      Top rated first
    </option>
  </select>
</form>
`;
